appveyor.yml

```diff
@@ -8,6 +8,16 @@ init:
 # Uncomment the following line to show RDP info at beginning of job
 #  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
 clone_folder: c:\projects\mixxx
+skip_commits:
+  files:
+    - doc/
+    - res/
```

Comment:
ok, I removed res/ because we don't need unit tests on fonts, translations & skins but we may need an installation package to manually test them.